src/linpred.jl

```diff
@@ -7,6 +7,11 @@ The effective coefficient vector, `p.scratchbeta`, is evaluated as `p.beta0 .+ f
 and `out` is updated to `p.X * p.scratchbeta`
 """
 function linpred!(out, p::LinPred, f::Real=1.)
+    for i in eachindex(p.delbeta)
+        if isnan(p.delbeta[i]) || isinf(p.delbeta[i])
+            p.delbeta[i] = 0
+        end
+    end
```

Comment:
Just in case the compiler isn't able to hoist the field access:
```suggestion
    delbeta = p.delbeta
    @inbounds for i in eachindex(delbeta)
        if isnan(delbeta[i]) || isinf(delbeta[i])
            delbeta[i] = 0
        end
    end
```

Reply:
This comment still applies.